testdriver/onboarding.test.mjs

```
@@ -118,29 +118,70 @@ describe("Wave Terminal onboarding", () => {
+            try {
+                const evidence = await testdriver.exec(
+                    "pwsh",
+                    `$log = Join-Path $env:LOCALAPPDATA "waveterm\\waveapp.log"\n` +
+                    `Write-Output "=== evidence: ${label} ==="\n` +
+                    `Write-Output "--- processes ---"\n` +
+                    `Get-Process -Name Wave,wavesrv -ErrorAction SilentlyContinue | Select-Object Name,Id | Format-Table | Out-String\n` +
+                    `Write-Output "--- waveapp.log tail ---"\n` +
+                    `if (Test-Path $log) { Get-Content $log -Tail 40 } else { Write-Output "(waveapp.log not found)" }\n`,
+                    60000
+                );
+                console.log(evidence);
```

ESLint

[error] 'console' is not defined.

(no-undef)